targets/lenovo-x1-carbon.nix

```diff
@@ -509,11 +509,12 @@
     };
     package = let inherit ((hostConfiguration.extendModules {modules = [formatModule];})) config; in config.system.build.${config.formatAttr};
   in {
-    inherit hostConfiguration package;
+    inherit package;
+    hostConfiguration = hostConfiguration.extendModules {modules = [disko.nixosModules.disko (import ../templates/targets/x86_64/generic/disk-config.nix)];} // {rawConfiguration = hostConfiguration;};
```

Comment:
Please do not import anything from templates, they are just merely meant to be templates when you say `nix flake new ...`